dask/dataframe/io/parquet/arrow.py

```diff
@@ -311,7 +311,13 @@ def write_partition(
         if index_cols:
             df = df.set_index(index_cols)
             preserve_index = True
-        t = pa.Table.from_pandas(df, preserve_index=preserve_index)
+
+        if 'schema' in kwargs:
+            schema = kwargs.pop('schema')
+            t = pa.Table.from_pandas(df, preserve_index=preserve_index, schema=schema)
+        else:
+            t = pa.Table.from_pandas(df, preserve_index=preserve_index)
```

Comment:
Would it make sense to add an explicit `schema=` keyword to the function definition and avoid this logic?

Reply:
that's not my understanding of how the parquet split-to-pyarrow-or-fastparquet works - you pass custom arbitrary keywords to the specific engines.

i really don't care how this works though or what the argument name is.

Reply:
Right.  The `schema=` keyword wouldn't be advertized on the top-level `dd.to_parquet` function, but it seems entirely reasonable to have it on the `ArrowEngine.write_partition` function.  At this point it's just an implementation detail.

Reply:
Like @mrockin said - I think it would make the most sense to just add `schema=None` as a keyword argument in the function definition, and then just add `schema=schema` in the call to `pa.Table.from_pandas`.

Otherwise, this seems fine to me - I had wanted to add an easy/clean way for users to pass these kinds of targeted kwargs in the refactor, but never got around to it.

Reply:
cool. will do.